data/migrations/20201109125611_create-hospital-and-amenities-tables.js

```diff
@@ -0,0 +1,24 @@
+exports.up = function(knex) {
+    return knex.schema
+    .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
+    .createTable('hospitals', function(table) {
+        table.integer('hospital_id');
+        table.string('hospital_name');
+        table.integer('amenities_id');
```

Comment:
I don't think we need the amenities_id as part of the hospital table. I would think we would need the closest_bridge_id instead (Something like that)